scrapers/stackoverflow.js

```diff
@@ -0,0 +1,61 @@
+var unirest = require("unirest");
+var question = "";
+for (i = 2; i < process.argv.length; i++) {
+    question += process.argv[i] + " ";
+}
+var url = "https://api.stackexchange.com/2.2/search/advanced?key=jAAMPr3uTSgYulMJ3LY5Hw((&order=desc&sort=activity&q=" + question + "&answers=1&site=stackoverflow";
```

Comment:
Github scraper - Uses GitHub api that has restrictions - https://developer.github.com/v3/rate_limit/

Instagram scraper - It scrapes instagram.com/. It isn't allowed it seems.
Check https://www.instagram.com/robots.txt

Similarly for quora scrapper  -  https://www.quora.com/robots.txt 
and timeanddate scrapper - https://www.timeanddate.com/robots.txt